tests/e2e/tests/orders/full-refund.spec.js

```diff
@@ -34,7 +34,7 @@ test( 'merchant can issue a full refund @smoke', async ( { browser } ) => {
 			config.get( 'cards.basic' )
 		);
 		await userPage.locator( 'text=Place order' ).click();
-		await userPage.waitForURL( '**/checkout/order-received/**' );
+		await userPage.waitForNavigation();
```

Comment:
`waitForNavigation()` is deprecated, and it may cause race conditions... using `waitForURL` is the recommended approach.

Reply:
It's not part of this PR; reverted it.